### DISCUSSION TITLE
Show last person to edit an entry on entry listing page

### DISCUSSION THREAD

**Discussion by agun** [ExpressionEngine/ExpressionEngine#2866] [Feature Requests]
**Describe the solution you'd like**
Currently on the entry listing page you can select to see the current entry author (this works, and is great) ... however it would also be good to be able to see the last person who edited the entry. You can see the last date/time the entry was edited, but seeing the person who did the editing would be great. 

It may be that you can only see this data when you have revisions turned on. That would also work okay in my scenario.

**Describe alternatives you've considered**
Currently you can see the last person who edited the entry using the revisions feature, but that requires going into each entry, clicking on the revisions tab, etc. Having it on the entry listing page would be great.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
This is nice idea, however those data are not available on Entry model.

I agree we could get those from Revisions table - though that would make queries more complex

**Comment by agun** [comment]
I agree. The only real solution would be to add in a new column into the exp_channel_titles table ... "edit_member_id" (or something like that). This wouldn't be a "log" of editors, but only the ID of the last person who edited the entry. That column would get updated the same as the "edit_date".

If you need a log of changes for the entry, that is what revisions is for.

**Comment by intoeetive** [comment]
Planning this for 7.6: https://github.com/ExpressionEngine/ExpressionEngine/pull/4838
